src/main/java/com/alkemy/ong/service/CategoryService.java

```diff
@@ -1,18 +1,22 @@
-package com.alkemy.ong.service.impl;
+package com.alkemy.ong.service;
 
 import com.alkemy.ong.dto.CategoryDto;
+import com.alkemy.ong.dto.CategoryDtoFull;
 import com.alkemy.ong.entity.CategoryEntity;
 import com.alkemy.ong.mapper.CategoryMapper;
 import com.alkemy.ong.repository.CategoryEntityRepository;
+import lombok.RequiredArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
 
 import java.util.List;
-
+@RequiredArgsConstructor
 @Service
 public class CategoryService {
 
-    private CategoryMapper categoryMapper;
+    private final CategoryMapper categoryMapper;
     @Autowired
     private CategoryEntityRepository categoryEntityRepository;
```

Comment:
should we refactor this repository to CategoryRepository? I think is simpler

Reply:
I think the same, I will raise it in the next meeting